### PR TITLE
Add URLs to web-vitals dashboard

### PR DESCRIPTION
**Pull request by nicola-d-bbc**
Resolves https://github.com/bbc/simorgh-infrastructure/issues/1467

**Overall change:**
Adds new pageTypes to include new types described in issue.

**Code changes:**

- Adds CPS_LONG_ARTICLE_WITH_PNG_AND_VISJO, CPS_LONG_ARTICLE_YOUTUBE_EMBED, CPS_MEDIUM_ARTICLE, CPS_SHORT_ARTICLE_YOUTUBE_EMBED, CPS_SHORT_ARTICLE, HOMEPAGE_SHORT, HOMEPAGE_LONG, MEDIA_ASSET_AUDIO, MEDIA_ASSET_VIDEO, RADIO_PAGE, PODCAST_PAGE to new pageTypes.js within web vitals.
- Added if statements to include new page types with given URLs.

---

- [x] I have assigned myself to this PR and the corresponding issues
- [ ] I have added the `cross-team` label to this PR if it requires visibility across World Service teams
- [x] I have assigned this PR to the Simorgh project
- [x] (BBC contributors only) This PR follows the [repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)

**Testing:**

- [x] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false yarn test:e2e:interactive`)
- [ ] This PR requires manual testing

### CODE REVIEW

File: src/app/routes/utils/pageTypes.js [hunk 1]
```diff
@@ -10,3 +10,17 @@ export const MEDIA_ASSET_PAGE = 'MAP';
...
+export const CPS_MEDIUM_ARTICLE = 'WS-CPS_MEDIUM_ARTICLE';
+export const CPS_SHORT_ARTICLE_YOUTUBE_EMBED =
+  'WS-CPS_SHORT_ARTICLE_YOUTUBE_EMBED';
+export const CPS_SHORT_ARTICLE = 'WS-CPS_SHORT_ARTICLE';
+export const HOMEPAGE_SHORT = 'WS-HOMEPAGE_SHORT';
+export const HOMEPAGE_LONG = 'WS-HOMEPAGE_LONG';
+export const MEDIA_ASSET_AUDIO = 'WS-MEDIA_ASSET_AUDIO';
+export const MEDIA_ASSET_VIDEO = 'WS-MEDIA_ASSET_VIDEO';
+export const RADIO_PAGE = 'WS-RADIO';
+export const PODCAST_PAGE = 'WS-PODCAST';
```
Comment (ryanmccombe) [inline src/app/routes/utils/pageTypes.js]: Looking at this in more detail, I'd probably suggest storing these new variables separately - eg somewhere within `containers/WebVitals`

The original set of variables have a different purpose - mixing the very specific web vital variables in here might confuse people in the future, especially given this file is in routing utilities.

**Review by ryanmccombe — APPROVED** [review]
👍

**Comment by ryanmccombe** [comment]
Looks like the unit tests are failing because the code now expects `RequestContext` to provide a `pathname` string, but it's not being passed in the tests

https://github.com/bbc/simorgh/blob/3e27c3af8ab17885a073dc677657d79b85ab5d63/src/app/containers/WebVitals/index.test.jsx#L32

File: src/app/containers/WebVitals/index.jsx [hunk 2]
```diff
@@ -2,16 +2,66 @@
...
+  if (pathname.includes('persian/podcasts/p0703hz7')) {
+    return PODCAST_PAGE;
+  }
+  return pageType;
+};
+
 const WebVitals = () => {
   const { personalisationEnabled } = useContext(UserContext);
-  const { pageType } = useContext(RequestContext);
+  const data = useContext(RequestContext);
```
Comment (ryanmccombe) [inline src/app/containers/WebVitals/index.jsx]:
```suggestion
  const requestContextData = useContext(RequestContext);
```

File: src/app/containers/WebVitals/index.jsx [hunk 3]
```diff
@@ -23,7 +73,7 @@ const WebVitals = () => {
     enabled: isWebVitalsEnabled,
     reportingEndpoint: process.env.SIMORGH_WEBVITALS_REPORTING_ENDPOINT,
     sampleRate,
-    reportParams: { pageType },
+    reportParams: { pageType: getPageType(data) },
```
Comment (ryanmccombe) [inline src/app/containers/WebVitals/index.jsx]:
```suggestion
    reportParams: { pageType: getPageType(requestContextData) },
```

**Review by amoore108 — APPROVED** [review]

**Pull request merged**